config/routes.rb

```diff
@@ -4,4 +4,8 @@
   resources :posts
 
   root to: redirect('/posts')
+
+  resources :users,
+    controller: 'users',
+    only: 'create'
```

Comment:
Personally I'd have this on one line, since it's only short, but it's not a big deal.